client/src/components/Sidebar/modules/Search.scss

```diff
@@ -1,16 +1,10 @@
 // stylelint-disable declaration-no-important
 .sidebar-search {
+  @apply w-z-20 w-relative w-box-border w-flex w-items-center w-flex-row w-h-[42px] w-px-5;
```

Comment:
Can you explain why this needs a `z-index`? I can’t see a difference either way from my testing. If it needs to stay this would warrant a comment.

Reply:
Ahh yes, this was because the tail of the logo was overlapping the search, however I was able to fix this now by adding overflow hidden to the logo until its hovered 🙌